QUnit/only.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: default
+title: only
+description: Adds a test to exclusively run, preventing all other tests from running.
+categories:
+  - main
+---
+
+## `QUnit.only( name, callback )`
+
+Adds a test to exclusively run, preventing all other tests from running.
+
+| parameter | description |
+|-----------|-------------|
+| `name` (string) | Title of unit being tested |
+| `callback` (function) | Function to close over assertions |
+
+#### Callback parameters: `callback( assert )`:
+
+| parameter | description |
+|-----------|-------------|
+| `assert` (object) | A new instance object with the [assertion methods](/api/assert) |
+
+### Description
+
+Use this method to focus your test suite on a specific test. `QUnit.only` will cause any other tests in your suite to be ignored.
+
+Note that if more than one `QUnit.only` is present only the first instance will run.
+
+This is an alternative to filtering tests to run in the HTML reporter. It is especially useful when you use a console reporter or in a codebase with a large set of long running tests.
+
+### Example
+
+How to use `QUnit.only` to filter your test suite.
+
+```js
+QUnit.module( "robot", {
+  beforeEach: function() {
+    this.robot = new Robot();
+  }
+});
+
+test( "say", function( assert ) {
```

Comment:
Would prefer this say `QUnit.test` unless a destructure is shown at the top of this sample.